.github/workflows/go_ci.yaml

```
@@ -0,0 +1,38 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: CI (Go)
+
+on:
+  pull_request:
+    branches: ["main", "development"]
+  push:
+    branches: ["main", "development"]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)